external-crates/move/crates/move-compiler/src/shared/mod.rs

```diff
@@ -958,15 +958,24 @@ impl Visitors {
         };
         for pass in passes {
             match pass {
-                Visitor::AbsIntVisitor(f) => vs.abs_int.push(RefCell::new(f)),
-                Visitor::TypingVisitor(f) => vs.typing.push(RefCell::new(f)),
-                Visitor::CFGIRVisitor(f) => vs.cfgir.push(RefCell::new(f)),
+                Visitor::AbsIntVisitor(f) => vs.abs_int.push(f),
+                Visitor::TypingVisitor(f) => vs.typing.push(f),
+                Visitor::CFGIRVisitor(f) => vs.cfgir.push(f),
             }
         }
         vs
     }
 }
 
+// TODO remove it once visitor invocation is parallel
+#[allow(unused)]
+fn check<T: Send + Sync>() {}
+#[allow(unused)]
+fn check_all() {
+    check::<Visitors>();
+    check::<&Visitors>();
+}
```

Comment:
The change is really centered around getting this check to pass. Does anyone know a better way of doing this?